crates/aranya-daemon/src/afc.rs

```diff
@@ -185,6 +194,45 @@ where
             .map_err(|err| anyhow!("unable to remove AFC channel: {err}"))
     }
 
+    /// Delete all channels.
+    pub(crate) async fn delete_channels(&self) -> Result<()>
+    where
+        E: Engine<CS = C>,
+    {
+        self.shm
+            .lock()
+            .await
+            .write
+            .remove_all()
+            .map_err(|err| anyhow!("unable to remove AFC channels: {err}"))
```

Comment:
These should each just use context, noticed there's some existing ones.

```suggestion
            .context("unable to remove AFC channels")
```